packages/icons/icons.json

```diff
@@ -1969,182 +1969,182 @@
     {
         "displayName": "Annotation",
         "iconName": "annotation",
-        "tags": "note, comment, edit,",
+        "tags": "note, comment, edit, , edit, write",
```

Comment:
```suggestion
        "tags": "note, comment, edit, write",
```

Reply:
@gabrieljroth could you please save me a bit of review time and do another pass on these edits to make sure you don't have empty tags or redundant words like this in the rest of this PR?